.pre-commit-config.yaml

```diff
@@ -11,6 +11,15 @@ repos:
         entry: bash -c 'if [ ! -f "$(dirname "$dir")"/frontend/src/locales/ar.json ]; then just frontend/run i18n; fi'
         language: system
         pass_filenames: false
+      - id: check-no-console-code-block
+        name: Check that the "console" block is not used in docs
+        # Check that we are not using the ```console block in our documentation
+        # -R recursively checks files
+        # -l suppresses normal input and only lists files with errors
+        # Print an error message if any files are found
+        entry: bash -c 'grep -Rl --include="*.md" '\''```console'\'' documentation/ && { printf "\n[ERROR] The \"bash\" syntax selection in the above files should be used in place of \"console\" in code blocks"; exit 1; } || exit 0'
```

Comment:
`--include` can be replaced with the `files` param on the hook to only accept Markdown. That way the hook won't even run unless a Markdown file has changed.